modules/rostests/apitests/gdi32/StretchBlt.c

```diff
@@ -0,0 +1,144 @@
+#include "wingdi.h"
+#include "winuser.h"
+#include "mmsystem.h"
+#include "wine/winternl.h"
+#include "wine/test.h"
+
+static void test_StretchBlt(void)
+{
+    HBITMAP bmpDst, bmpSrc;
+    HBITMAP oldDst, oldSrc;
+    HDC hdcScreen, hdcDst, hdcSrc;
+    UINT32 *dstBuffer, *srcBuffer;
+    BITMAPINFO biDst, biSrc;
+    UINT32 expected[256];
+
+    memset(&biDst, 0, sizeof(BITMAPINFO));
+    biDst.bmiHeader.biSize = sizeof(BITMAPINFOHEADER);
+    biDst.bmiHeader.biWidth = 16;
+    biDst.bmiHeader.biHeight = -16;
+    biDst.bmiHeader.biPlanes = 1;
+    biDst.bmiHeader.biBitCount = 32;
+    biDst.bmiHeader.biCompression = BI_RGB;
+    memcpy(&biSrc, &biDst, sizeof(BITMAPINFO));
+
+    hdcScreen = CreateCompatibleDC(0);
+    hdcDst = CreateCompatibleDC(hdcScreen);
+    hdcSrc = CreateCompatibleDC(hdcDst);
+
+    bmpDst = CreateDIBSection(hdcScreen, &biDst, DIB_RGB_COLORS, (void**)&dstBuffer,
+        NULL, 0);
+    oldDst = SelectObject(hdcDst, bmpDst);
+
+    bmpSrc = CreateDIBSection(hdcScreen, &biSrc, DIB_RGB_COLORS, (void**)&srcBuffer,
+        NULL, 0);
+    oldSrc = SelectObject(hdcSrc, bmpSrc);
+
+    SelectObject(hdcSrc, oldSrc);
+    DeleteObject(bmpSrc);
+    SelectObject(hdcDst, oldDst);
+    DeleteObject(bmpDst);
+
+    memset(&biDst, 0, sizeof(BITMAPINFO));              // Clear our Bitmap Into to all zeroes
+
+    biDst.bmiHeader.biSize = sizeof(BITMAPINFOHEADER);  // Set our Size to the header size
+    biDst.bmiHeader.biWidth = 2;                        // Set our Width to 2
+    biDst.bmiHeader.biHeight = 2;                       // Set our Height to 2 that's positive (bottom-up)
+    biDst.bmiHeader.biPlanes = 1;                       // Set out planes to 1 (1 required by Windows)
+    biDst.bmiHeader.biBitCount = 32;                    // Set out BitCount to 32 (Full Color)
+    biDst.bmiHeader.biCompression = BI_RGB;             // Set our Compression to BI_RBG (uncompressed)
+
+    memcpy(&biSrc, &biDst, sizeof(BITMAPINFO));         // Put same Destination params into the Source
+
+    bmpSrc = CreateDIBSection(hdcSrc, &biSrc, DIB_RGB_COLORS, (void**)&srcBuffer, NULL, 0);
+    oldSrc = SelectObject(hdcSrc, bmpSrc);
+    bmpDst = CreateDIBSection(hdcDst, &biDst, DIB_RGB_COLORS, (void**)&dstBuffer, NULL, 0);
+    oldDst = SelectObject(hdcDst, bmpDst);
+
+    srcBuffer[0] = 255;             // BLUE// stored beginning bottom left
+    srcBuffer[1] = 255*256;         // GREEN
+    srcBuffer[2] = 255*256*256;     // RED
+    srcBuffer[3] = 127*256*256*256; // BLACK
+
+    /* Flip Left to Right on Source */
+    StretchBlt(hdcDst, 0, 0, 2, 2, hdcSrc, 1, 0, -2, 2, SRCCOPY );
+    expected[0] = 255*256;
+    expected[1] = 255;
+    expected[2] = 127*256*256*256;
+    expected[3] = 255*256*256;
+
+    ok(expected[1] == dstBuffer[1], "StretchBlt expected { %08X } got { %08X }\n",
+        expected[1], dstBuffer[1] );
+
+    ok(expected[3] == dstBuffer[3], "StretchBlt expected { %08X } got { %08X }\n",
+        expected[3], dstBuffer[3] );
```

Comment:
No space needed before `)`
```suggestion
    ok(expected[1] == dstBuffer[1], "StretchBlt expected { %08X } got { %08X }\n",
        expected[1], dstBuffer[1]);

    ok(expected[3] == dstBuffer[3], "StretchBlt expected { %08X } got { %08X }\n",
        expected[3], dstBuffer[3]);
```

Reply:
Fixed. Thanks.